man/systemd.network.xml

```diff
@@ -991,6 +991,13 @@
             Supported only on IPv6. Defaults to false.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>IPv4DuplicateAddressDetection=</varname></term>
```

Comment:
is yet another option really needed? we already have `DuplicateAddressDetection=`, currently only used for IPv6. It could be generalized to apply to both address families: use IPv4 ACD if the address is v4, use IPv6 DAD if the address is v6.

Reply:
Sounds nice. Thanks. I'd like to update so. @ssahani Do you agree with that?

Reply:
I noticed that `DuplicateAddressDetection=` is controversial. If set to yes, then `NODAD` flag is set. Maybe, we should deprecate the setting, and introduce `IPv6DuplicateAddressDetection=`.

Reply:
Yes . It works for IPv6. We set this and kernel does the DAD. Mixing up will not work. Renaming indeed is nice.

Reply:
oh, I didn't realize `DuplicateAddressDetection=` logic was inverted. Definitely surprising and user-unfriendly.

Unfortunately, there is already a `IPv6DuplicateAddressDetection=` option in the `[Network]` section. Could it be a source of confusion to have the same option name in different sections with different semantics?